scripts/check_frontend_coverage.py

```diff
@@ -313,6 +313,7 @@
     'oppia-short-response-set-input.directive.ts',
     'oppia-short-response-text-input.directive.ts',
     'oppia-visualization-bar-chart.directive.ts',
+    'oppia-visualization-click-hexbins.directive.ts',
```

Comment:
We are not expecting to add new files in this list, can we add tests for this new directive?

Reply:
That's correct, I had the same question in this [thread](https://github.com/oppia/oppia/pull/9324#pullrequestreview-416539669) but I've totally missed in this PR

Reply:
Can we add tests for this file too? Other than that LGTM as a codeowner!

Reply:
Done.